h2o-py/tests/testdir_algos/xgboost/pyunit_checkpoint_xgboost.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import print_function
+from builtins import range
+import sys
+sys.path.insert(1,"../../../")
+import h2o
+from tests import pyunit_utils
+import os
+import random
+from h2o.estimators.xgboost import H2OXGBoostEstimator
+from h2o.exceptions import H2OResponseError
+ 
+def xgboost_checkpoint():
+
+  milsong_train = h2o.upload_file(pyunit_utils.locate("bigdata/laptop/milsongs/milsongs-train.csv.gz"))
```

Comment:
is it possible to make a test that will make a model of say 50 trees and then train a model with 25, resume from checkpoint and train another 25 and make sure the predictions are the same?